src/Microsoft.Health.Dicom.Core/Extensions/CustomTagEntryExtensions.cs

```diff
@@ -0,0 +1,53 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Globalization;
+using Dicom;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Features.Common;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+
+namespace Microsoft.Health.Dicom.Core.Extensions
+{
+    /// <summary>
+    /// Extension methods for <see cref="CustomTagEntry"/>.
+    /// </summary>
+    internal static class CustomTagEntryExtensions
+    {
+        /// <summary>
+        /// Normalize custom tag entry before saving to CustomTagStore.
+        /// </summary>
+        /// <param name="customTagEntry">The custom tag entry.</param>
+        /// <returns>Formalized custom tag entry.</returns>
+        internal static CustomTagEntry Normalize(this CustomTagEntry customTagEntry)
```

Comment:
>internal [](start = 8, length = 8)

Typically I write the class's access modifiers as if the class were `public`, and then adjust the class's modifier as appropriate. So in this case, I think it's ok to make these `public`